src/Workspaces/Core/Portable/Storage/PersistenceStorageServiceFactory.cs

```diff
@@ -34,26 +34,13 @@ public IWorkspaceService CreateService(HostWorkspaceServices workspaceServices)
                 case StorageDatabase.SQLite:
                     var locationService = workspaceServices.GetService<IPersistentStorageLocationService>();
                     if (locationService != null)
-                    {
-                        if (UseInMemoryWriteCache(workspaceServices))
-                        {
-                            return new SQLite.v2.SQLitePersistentStorageService(locationService);
-                        }
-                        else
-                        {
-                            return new SQLite.v1.SQLitePersistentStorageService(locationService);
-                        }
-                    }
+                        return new SQLite.v2.SQLitePersistentStorageService(locationService);
```

Comment:
Do you want to get rid of the 'v2' namespace while you're at it too?

Reply:
(this is optional and not part of the block)

Reply:
i actually like it :)   esp. as this read/writes sxs with teh original v1 code.